aip/general/0193/aip.md

```diff
@@ -0,0 +1,159 @@
+# Errors
+
+Error handling is an important part of designing simple and intuitive APIs.
+Consistent error handling allows developers to know how to expect to receive
+errors, and to reduce boilerplate by having common error-handling logic, rather
+than being expected to constantly add verbose error handling everywhere.
+
+## Guidance
```

Comment:
since there already is protobuf-level guidance, why not include protobuf? or does it translate? https://google.aip.dev/193.

I'm thinking about how to reconcile this proposal with existing practices in google.aip.dev (which, as the only public repository up until now, has served as the base of other forks IIUC).